var/spack/repos/builtin/packages/openblas/test_cblas_dgemm.c

```diff
@@ -0,0 +1,14 @@
+#include <cblas.h>
+#include <stdio.h>
+int main(void) {
+int i=0;
+double A[6] = {1.0, 2.0, 1.0, -3.0, 4.0, -1.0};
+double B[6] = {1.0, 2.0, 1.0, -3.0, 4.0, -1.0};
+double C[9] = {.5, .5, .5, .5, .5, .5, .5, .5, .5};
+cblas_dgemm(CblasColMajor, CblasNoTrans, CblasTrans,
+            3, 3, 2, 1, A, 3, B, 3, 2, C, 3);
+for (i = 0; i < 8; i++)
+  printf("%lf ", C[i]);
```

Comment:
It's common to just write `%f` to output a double-precision value. (`scanf` and `printf` are not symmetric this way.)